### ISSUE TITLE
Don't lose reputation for stealing?

### ISSUE THREAD

**Issue by hubismal** [Regalis11/Barotrauma#3209]
[Bug?]
Repeatedly stealing stuff in the outpost. When my crew was eventually killed and the round summary screen displayed for the outpost, I only lost 1 reputation, presumably from hitting security with a crowbar. I am assuming this is a bug since stealing is definitely something you would lose reputation for, as implied by the loading screen.

**Comment by Rokvach** [comment]
If you don't get caught stealing there's nobody to lower your reputation, but if your reputation doesn't lower when you do get caught then that is is something which should be implemented.

**Comment by Regalis11** [comment]
Reputation loss from stealing had not been implemented at all, fixed in https://github.com/Regalis11/Barotrauma-development/commit/dccbdfc353fb710ba1124c5e957f76a049cd9796

**Comment by hubismal** [comment]
To clarify, yeah, I was caught stealing over and over, literally everything I stole I intentionally did in front of others.

**Comment by Rokvach** [comment]
It seems that stealing an item only lowers your reputation once per item, so if you already took the item out of the cabinet and put it back, your reputation doesn't lower once you take it out again. Is this intentional? @Regalis11

**Comment by Regalis11** [comment]
Yeah, that's intentional, otherwise it'd be easy to destroy the reputation.

**Comment by Rokvach** [comment]
No issues, closing.
